charts/sysdig/templates/_helpers.tpl

```diff
@@ -141,6 +141,33 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
 
+{{/*
+Daemonset labels
+*/}}
+{{- define "daemonset.labels" -}}
+  {{- if .Values.daemonset.labels }}
+    {{- $tp := typeOf .Values.daemonset.labels }}
+    {{- if eq $tp "string" }}
+      {{- tpl .Values.daemonset.labels . }}
```

Comment:
@jcbbc out of curiosity, what kind of values are you trying to template in the labels? would yaml anchors on in your values file solve it?

if there are useful values that we aren't labeling today, it might be worth adding them to the common label set

Reply:
Our primary use case is for DataDog monitoring, so these labels are very specific to our context.
```
tags.datadoghq.com/service: {{ .Release.Name | lower | quote }}
tags.datadoghq.com/version: {{ .Values.image.tag | quote }}
```